sigs/sig-marketing/README.md

```diff
@@ -0,0 +1,17 @@
+# Argo Sig Marketing
+
+Argo Sig Marketing coordinates efforts between interested parties to promote, advocate, and grow the Argo Project and community. This includes social media, subreddits, events, branding, and all other marketing activities.
+
+## Meetings
+Sig marketing meetings are held twice monthly. Meetings can be found in the [Argo Project public calendar](https://calendar.google.com/calendar/embed?src=argoproj%40gmail.com&ctz=America%2FDenver).
+
```

Comment:
```suggestion
## Membership Requirements 
- Must represent the open-source community’s best interest with a vendor-neutral practice.
- Must follow the CNCF rules and guidances closely.
- Must have shown a proven record of enabling the open-source community by marketing.
- Technical marketing is preferred.
- Cannot have more than two members from the same company.
- The membership will be revisited every calendar year.
```

Reply:
I believe that membership for all our SIGs should be open to anyone and everyone who wants to engage with a particular SIG. The SIGs are really working groups with a given charter and IMO, there's really no need to institute election/voting/membership procedures for them. All decisions should be consensus driven, and if consensus cant be reached, the maintainer group has the final say, as per our governance.